plugins/plugin-optimize/README.md

```diff
@@ -29,10 +29,12 @@ Then add this plugin to your Snowpack config:
-| `minifyHTML`     |     `boolean`     | Enable HTML minification (default: `true`)                                                                                                                                                                                                                     |
-| `preloadModules` |     `boolean`     | Experimental: Add deep, optimized [`<link rel="modulepreload">`](https://developers.google.com/web/updates/2017/12/modulepreload) tags into your HTML. (default: `false`)                                                                                      |
-| `target`         | `string,string[]` | The language target(s) to transpile to. This can be a single string (ex: "es2018") or an array of strings (ex: ["chrome58","firefox57"]). If undefined, no transpilation will be done. See [esbuild documentation](https://github.com/evanw/esbuild) for more. |
+| Name               |       Type        | Description                                                                                                                                                                                                                                                    |
+| :----------------- | :---------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `minifyJS`         |     `boolean`     | Enable JS minification (default: `true`)                                                                                                                                                                                                                       |
+| `minifyCSS`        |     `boolean`     | Enable CSS minification (default: `true`)                                                                                                                                                                                                                      |
+| `minifyHTML`       |     `boolean`     | Enable HTML minification (default: `true`)                                                                                                                                                                                                                     |
+| `preloadModules`   |     `boolean`     | Experimental: Add deep, optimized [`<link rel="modulepreload">`](https://developers.google.com/web/updates/2017/12/modulepreload) tags into your HTML. (default: `false`)                                                                                      |
```

Comment:
to match above naming, should this be `preloadJS`? Since there's no such thing as a non-module JS script "import"/"dependency", I don't think you need to worry about that distinction.